### DISCUSSION TITLE
API Jadwal KRL

### DISCUSSION THREAD

**Discussion by ghamenk** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#322] [General]
kak adakah API jadwal KRL?

**Comment by farizdotid** [comment]
Untuk daerah mana mas? yang saya tau di Jakarta sepertinya ada ( https://smartcity.jakarta.go.id/blog/343/cara-pemanfaatan-api-jakarta-smart-city )

**Comment by ghamenk** [comment]
untuk KRL saya belum nemu kak kalau busway ada.. jadi saya sedang tugas akhir kuliah mau buat info kedatangan KRL untuk stasiun rawabuntu tangerang

**Comment by farizdotid** [comment]
Coba kolaborasi dengan Dinas terkait mas
